src/Login/actions.js

```diff
@@ -0,0 +1,47 @@
+import LOGIN from './action_types';
+import query from '../util/query';
+
+export function setUsername(value) {
+    return {
+        type: LOGIN.USERNAME,
+        value
+    };
+}
+
+export function setPassword(value) {
+    return {
+        type: LOGIN.PASSWORD,
+        value
+    };
+}
+
+export function toggleRemember(value) {
+    return {
+        type: LOGIN.REMEMBER,
+        value
+    };
+}
+
+export function login() {
+    return {
+        type: LOGIN.SUBMIT
+    };
+}
+
+export function attemptLogin(uid, password) {
```

Comment:
We should try to include everything into GraphQL, and use one API manager to handle the outgoing request. 

Eventually, we want to API manager to be smart enough to decide if we want to send a GraphQL endpoint or all `API` based request.